src/components/map/Map.tsx

```diff
@@ -26,27 +28,32 @@ const MARKER_ICON_SIZE = normalize(40);
 
 export const Map = ({
   geometryTourData,
-  isMaximizeButtonVisible,
+  isMaximizeButtonVisible = false,
   isMultipleMarkersMap = false,
   locations,
   mapCenterPosition,
   mapStyle,
   onMapPress,
   onMarkerPress,
   onMaximizeButtonPress,
+  showPOIsFullScreenMap,
   style,
   ...otherProps
 }: Props) => {
   const { globalSettings } = useContext(SettingsContext);
   const { settings = {} } = globalSettings;
   const { zoomLevelForMaps = {}, locationService = {} } = settings;
 
+  // state to keep track of the currently active marker
+  const [activeMarker, setActiveMarker] = useState(null);
+
   const showsUserLocation = otherProps.showsUserLocation ?? !!locationService;
   const zoom = isMultipleMarkersMap
     ? zoomLevelForMaps.multipleMarkers
     : zoomLevelForMaps.singleMarker;
```

Comment:
you wrote

> - added `zoomLevelForMaps` and `locationService` from settings to the `Map` component for
  configurable zoom and location features

but this was already implemented, right?!